src/utils/utils.ts

```diff
@@ -72,4 +72,25 @@ export const utils = {
         }
         return chunkedItems;
     },
+    convertAmountToBigNumber(value: string | number | BigNumber): BigNumber {
+        const num = value || 0;
+        const isBigNumber = BigNumber.isBigNumber(num);
+        if (isBigNumber) {
+            return num as BigNumber;
+        }
+
+        if (_.isString(num) && (num.indexOf('0x') === 0 || num.indexOf('-0x') === 0)) {
```

Comment:
when is `-0x` valid?

Reply:
Good question, this was some copy-pasta from `web3-wrapper`, which in turn was copy-pasta'd from `ethereumjs-utils`. I've gone ahead and imported it from the `web3-wrapper` package instead of copying this code once again.